test/splitter.js

```diff
@@ -33,24 +32,24 @@ contract('Splitter', async accounts => {
 
         // Take snapshot of initial balances
         let initSenderEthBalance = await web3.eth.getBalance(senderAddress);
-        initSenderEthBalance = web3.utils.toBN(initSenderEthBalance);
+        initSenderEthBalance = toBN(initSenderEthBalance);
 
         const initSenderContractBalance = await splitter.balances(senderAddress);
-        const initRecipientOneContractBalance = await splitter.balances(RecipientOneAddress);
-        const initRecipientTwoContractBalance = await splitter.balances(RecipientTwoAddress);
+        const initRecipientOneContractBalance = await splitter.balances(recipientOneAddress);
+        const initRecipientTwoContractBalance = await splitter.balances(recipientTwoAddress);
 
         // Perform transactions
-        const receipt = await splitter.splitDeposit(RecipientOneAddress, RecipientTwoAddress, {from: senderAddress, value: 5});
+        const txObj = await splitter.splitDeposit(recipientOneAddress, recipientTwoAddress, {from: senderAddress, value: 5});
 
-        const cost = await gasCost(receipt);
-        const transfer = web3.utils.toBN(5);
-        const splitAmount = web3.utils.toBN(2);
-        const remainder = web3.utils.toBN(1);
+        const cost = await getGasCost(txObj);
+        const transfer = toBN(5);
+        const splitAmount = toBN(2);
+        const remainder = toBN(1);
 
         // Check sender's changed ETH balance
-        const expectedSenderEthBalance = initSenderEthBalance.sub(cost).sub(web3.utils.toBN(transfer));
+        const expectedSenderEthBalance = initSenderEthBalance.sub(cost).sub(toBN(transfer));
         let aliceEthBalance = await web3.eth.getBalance(senderAddress);
-        aliceEthBalance = web3.utils.toBN(aliceEthBalance);
+        aliceEthBalance = toBN(aliceEthBalance);
```

Comment:
Instead of `let` and overwrite it, you can do `const aliceEthBalance = toBN(await web3.eth.getBalance(senderAddress));`